### ISSUE TITLE
Discrepancies between Insights XB data & Dashboard data

### ISSUE THREAD

**Issue by gemplank** [humanmade/aws-analytics#378]
On the main Altis site, we've got an A/B block running on a campaign page and it appears that the page views listed in the dashboard doesn't correlate to the number of 'block' views within the Insights area. The block is above the fold so should be triggering a view... Page views for example is 34 over 7 days, yet Insights thinks that block views is 9. Can provide URL's to whoever picks this ticket up (public repo).

TL;DR doesn't seem that Insights is showing the correct statistics
